source/stylesheets/refills/_phone.scss

```diff
@@ -0,0 +1,68 @@
+.phone {
+  $medium-screen: em(640) !default;
+  $large-screen: em(860) !default;
+  $ratio-small: 1;
+  $ratio-large: 1.6;
+  $inset-shadow: inset 0 1px 4px transparentize(#000, 0.8);
+  $phone-padding: 0.4em;
+  $phone-screen-width: 8.5em;
+  $phone-screen-height: 15em;
+  $phone-background: #F8F8F8;
+
+  background: $phone-background;
+  border-radius: (1.2em * $ratio-small);
+  box-shadow: 0 0 1px transparentize(#000, 0.4), inset 0 3px 0 #fff;
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.